x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -604,7 +605,9 @@ const StatefulEventsViewerComponent: React.FC<EventsViewerProps & PropsFromRedux
                             pagination={pagination}
                             isEventRenderedView={tableView === 'eventRenderedView'}
                             rowHeightsOptions={rowHeightsOptions}
-                            getFieldBrowser={getFieldBrowser}
+                            getFieldBrowser={
+                              getFieldBrowser as unknown as DataTableProps['getFieldBrowser']
```

Comment:
Why is this casting needed?

Reply:
I will try to fix it some other way:) seems like the data field browser thing also uses some custom field structure, same as we used to have till now.